### ISSUE TITLE
Unused function?

### ISSUE THREAD

**Issue by squigglybob** [prykon/pray4movement-prayer-points#6]
https://github.com/prykon/pray4movement-prayer-points/blob/1ebafdb9469fc7f24972ce80a78817bf36bdbc28/admin/admin-menu-and-tabs.php#L633

FYI, this code doesn't seem to be used anywhere in the plugin.

Also it should also probably return true if it's all good as well

**Comment by prykon** [comment]
Good catch! This function will be removed in the next commit. I had to move it to an inline function to satisfy PHPCS.
